IngressMaxFields.user.js

```diff
@@ -68,74 +70,96 @@ function wrapper() {
-						var str1 = p.options.data.title.replace(/\"/g, "\\\"");
-						var str2 = str1.replace(';', ' ');
+		}
+		return obj;
 
-						//check for keys plugin
-						if (window.plugin.keys) {
-							var keyCount = window.plugin.keys.keys[x] || 0;
-							o.push(str2 + ";" + href + ";" + keyCount);
-						} else {
-							o.push(str2 + ";" + href);
-						}
-					}
-				} else {
-					if (tooMany == 0) {
-						tooMany = 1;
-					}
-				}
+	};
+	self.checkPortals= function checkPortals(portals) {
+		var obj = {
+			list:[],
+			tooMany:false,
+			count:0
+			};
+		for (var x in portals) {
+			if (typeof window.portals[x] !== "undefined") {
+				self.managePortals(obj,window.portals[x],x);
 			}
+		}
+		obj.tooMany = obj.count < 50?false:true;
+		return obj;
+
 
-			var dialog = window
-				.dialog({
-					title: "www.ingress-maxfield.com: Field your future",
-					// body must be wrapped in an outer tag (e.g.
-					// <div>content</div>)
-					html: '<span>Save the data in a textfile or post it on ingress-maxfields.com. ' + string + '</span>' + '<form name="maxfield" action="http://ingress-maxfield.com/submit.php" enctype="multipart/form-data" method="post" target="_blank_"><textarea name="portal_list_area" id="upload" rows="30" style="width: 100%;"></textarea><p>Number of agents:<input type="number" class="num_agents" name="num_agents" value="1" min="1" required></p><p>Use Google maps<input type="checkbox" name="useGoogle" value="YES" checked><input type="hidden" name="email" placeholder="(optional)"></p><p><input type="submit" class="submit" name="submit" value="Submit!"></p></form>'
+	};
+	self.showDialog = function showDialog(o,tooMany)
+	{
+		var data = "<span>Save the data in a textfile or post it on ingress-maxfields.com.</span>";
+		data = data + "<form name=\"maxfield\" action=\"http://ingress-maxfield.com/submit.php\" enctype=\"multipart/form-data\" method=\"post\" target=\"_blank\">";
+		data = data + "<textarea name=\"portal_list_area\" id=\"upload\" rows=\"30\" style=\"width: 100%;\">" + o.join("\n") + "</textarea>";
+		data = data + "<p>Number of agents:<input type=\"number\" class=\"num_agents\" name=\"num_agents\" value=\"1\" min=\"1\" required></p>";
+		data = data + "<p>Use Google maps<input type=\"checkbox\" name=\"useGoogle\" value=\"YES\" checked>";
+		data = data + "<input type=\"hidden\" name=\"email\" placeholder=\"(optional)\"></p><p><input type=\"submit\" class=\"submit\" name=\"submit\" value=\"Submit!\">";
+		data = data + "</p></form>";
+		var dia = window
+			.dialog({
+				title: "www.ingress-maxfield.com: Field your future",
+					html: data
 				}).parent();
 			$(".ui-dialog-buttonpane", dialog).remove();
-			dialog.css("width", "600px").css("top",
-				($(window).height() - dialog.height()) / 2).css("left",
-				($(window).width() - dialog.width()) / 2);
-			$("#upload").val(o.join("\n"));
-			if (tooMany == 1) {
+			dia.css("width", "600px").css("top",
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: ["$" is not defined.](https://www.codacy.com/app/stefan-nygren/IITC-Ingress-Maxfields-Exporter/file/1752266469/issues/source?bid=3039347&fileBranchId=3045939#l142)